src/enforcer.js

```diff
@@ -610,16 +712,16 @@ export class TrustedTypesEnforcer {
    * @private
    */
   wrapSetter_(object, name, type, descriptorObject = undefined) {
-    if (descriptorObject && !isPrototypeOf.call(descriptorObject, object)) {
+    if (!this.windowObject.Node) return;
```

Comment:
I'd prefer if we didn't call the function when Node is not available instead.

Reply:
Done.